okta-spring-boot-starter/src/test/groovy/com/okta/spring/oauth/http/UserAgentTest.groovy

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2014 Stormpath, Inc.
+ * Modifications Copyright 2018 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.spring.oauth.http
+
+import org.springframework.util.StringUtils
+import org.testng.annotations.Test
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.*
```

Comment:
Ah! A wild `*` appeared.

Reply:
I'm a fan of the wildcard for hamcrest Matchers

Reply:
Isn't it just `allOf`, `not`, `containsString`, and `emptyString`? Seems like we should pollute the scope when we're exposing < 10 matchers.

Reply:
It is just a test though, it makes it easier to write tests as type completion works faster (and those methods do not conflict with any other static imports)

This is not the case when using some libs (IIRC easymock and hamcrest conflict)

Reply:
It looks like there is a 50/50 split when it comes to wildcard imports with `hamcrest.Matchers`. So i'll let you make the final decision here.